src/components/TargetForm/TargetFormContent.js

```diff
@@ -69,65 +72,66 @@ const TargetFormContent = ({
     topicsError,
     loadingTopics,
     topicPlaceholder,
+    visible,
+    initialValues,
   });
 
+  const onEndEditingArea = () => {
+    setValues({
+      ...values,
+      areaLength: values.areaLength ? `${values.areaLength} m` : initialValues.areaLength,
+    });
+  };
+
   return (
     <>
       <Input
         label={existent ? areaEdit : area}
         testID="area-input"
         keyboardType="numeric"
-        error={errors[FIELDS.areaLength]}
-        invalid={targetError || errors[FIELDS.areaLength]}
         short={false}
         additionalStyles={styles.inputContainer}
-        placeholder={areaPlaceholder}
-        placeholderTextColor={BLACK}
         onBlur={areaInputProps.onBlur}
         onChangeText={areaInputProps.onChangeText}
-        value={existent ? existent.radius.toString() : areaInputProps.value}
+        value={existent ? `${existent.radius.toString()} m` : areaInputProps.value}
```

Comment:
I see that you are repeating the string format too many times consider making a helper function to not repeat that part to much